src/registry_schemas/schemas/continuation_in.json

```diff
@@ -10,14 +10,38 @@
     "properties": {
         "continuationIn": {
             "type": "object",
-            "required": [
-                "foreignJurisdiction",
-                "authorization",
-                "nameRequest",
-                "offices",
-                "parties"
-            ],
+            "if": {
+                "properties": {
+                    "isApproved": {
+                        "const": false
+                    }
+                }
+            },
+            "then": {
+                "$comment": "if isApproved is false or not defined",
+                "required": [
+                    "nameRequest",
+                    "authorization",
+                    "foreignJurisdiction",
+                    "contactPoint"
+                ]
+            },
+            "else": {
+                "required": [
+                    "nameRequest",
+                    "authorization",
+                    "foreignJurisdiction",
+                    "contactPoint",
+                    "offices",
+                    "parties",
+                    "shareStructure"
+                ]
+            },
             "properties": {
+                "isApproved": {
```

Comment:
Setting this value to `True` will be handled from legal-api (before validating filing json)

Reply:
So the UI will need to read this from a draft and then save it upon filing?

Reply:
Oh, so Legal API will set this before validating the schema?

Reply:
Does it have to be a declared property in this schema? I thought our schemas allowed additional properties without checks.

Reply:
As you can see from line no. 13 to 39 we have an "if -> then -> else", which is based on this field

Reply:
OK. Then this is fine by me.